misc/systemd/ipfs.service

```diff
@@ -1,9 +1,29 @@
 [Unit]
-Description=IPFS Daemon
+Description=InterPlanetary File System (IPFS) daemon
+After=network.target
 
 [Service]
+
+# enable for 1-1024 port listening
+#AmbientCapabilities=CAP_NET_BIND_SERVICE 
+# enable to specify a custom port
+#Environment=IPFS_PATH=/custom/ipfs/path
+# enable to specify a higher limit for open files/connections
+#LimitNOFILE=1000000
+# enable to specify a higher swarm FD-Limit
+#Environment=LIBP2P_SWARM_FD_LIMIT=800
+# enable to enforce a memory limit
+#MemoryLimit=5000M
+# enable to specify the preferenced connection muxer
+#Environment=LIBP2P_MUX_PREFS="/mplex/6.7.0 /yamux/1.0.0 /spdy/3.1.0"
+# enable to specify a limit for I/O load while starting
+#StartupIOWeight=10
+# enable to specify a limit for I/O load while running
+#IOWeight=50
```

Comment:
I would drop everything but the `AmbientCapabilities` and `LimitNOFILE` lines. The environment variables are better documented in docs/environment-variables.md and the rest of these are just general systemd parameters.

Reply:
How about changing it to point to the docs/environment-variables.md for the environment variables, but keep one as an example (the path - since this is probably the most common).

The general systemd parameters are just an example of useful resource limitations for such a daemon - sure, I can remove them if you think that's not helpful.

Reply:
SGTM.

> The general systemd parameters are just an example of useful resource limitations for such a daemon - sure, I can remove them if you think that's not helpful.

I'd leave them out. systemd has a ton of tunables and these aren't all that specific to go-ipfs (except LimitNOFILE and AmbientCapabilities).